src/backend/distributed/master/master_metadata_utility.c

```diff
@@ -57,6 +61,271 @@ static void RecordDistributedRelationDependencies(Oid distributedRelationId,
 												  Node *distributionKey);
 static ShardPlacement * TupleToShardPlacement(TupleDesc tupleDesc,
 											  HeapTuple heapTuple);
+static uint64 DistributedTableSize(Oid relationId, char *sizeQuery);
+static uint64 DistributedTableSizeOnWorker(WorkerNode *workerNode, Oid relationId,
+										   char *sizeQuery);
+static List * ShardIntervalsOnWorkerNode(WorkerNode *workerNode, Oid relationId);
+static StringInfo GenerateSizeQueryOnMultiplePlacements(Oid distributedRelationId,
+														List *shardIntervalList,
+														char *sizeQuery);
+static void ErrorIfNotSuitableToGetSize(Oid relationId);
+
+
+/* exports for SQL callable functions */
+PG_FUNCTION_INFO_V1(citus_table_size);
+PG_FUNCTION_INFO_V1(citus_total_relation_size);
+PG_FUNCTION_INFO_V1(citus_relation_size);
+PG_FUNCTION_INFO_V1(master_stage_shard_row);
```

Comment:
As long as I see, these functions are dropped in a previous commit: https://github.com/citusdata/citus/commit/a2ba565a9e1835fc82fc2a21268f011046de4638

Could you remove them?